### PR TITLE
[INLONG-9920][Audit] Audit-service add codes of source

### LINKED ISSUE #9920: [Feature][Audit] Audit-service add codes of source
### Description

Audit-service add codes of source

### Use case

Audit-service add codes of source

### Are you willing to submit PR?

- [X] Yes, I am willing to submit a PR!

### Code of Conduct

- [X] I agree to follow this project's [Code of Conduct](https://www.apache.org/foundation/policies/conduct)

### PR DESCRIPTION
**Pull request by doleyzi**
### Prepare a Pull Request

- Fixes #9920

### Motivation

*Add codes of source in audit-service module to real-time statistics of data from clickhouse, mysql and other data sources*

### Modifications

*Audit-service add codes of source.*

### CODE REVIEW

File: inlong-audit/audit-service/src/main/java/org/apache/inlong/audit/selector/impl/DBDataSource.java [hunk 1]
```diff
@@ -29,28 +32,17 @@
...
-    private String selectorSql = SELECTOR_SQL;
-    private String replaceLeaderSql = REPLACE_LEADER_SQL;
-    private String reLeaseSql = RELEASE_SQL;
-    private String isLeaderSql = IS_LEADER_SQL;
-    private String searchCurrentLeaderSql = SEARCH_CURRENT_LEADER_SQL;
+    private String selectorSql = SqlConstants.SELECTOR_SQL;
+    private String replaceLeaderSql = SqlConstants.REPLACE_LEADER_SQL;
+    private String reLeaseSql = SqlConstants.RELEASE_SQL;
+    private String isLeaderSql = SqlConstants.IS_LEADER_SQL;
+    private String searchCurrentLeaderSql = SqlConstants.SEARCH_CURRENT_LEADER_SQL;
```
Comment (aloyszhang) [inline inlong-audit/audit-service/src/main/java/org/apache/inlong/audit/selector/impl/DBDataSource.java:45]: Using the constants from `SqlConstants` is enough, no need to create new variables

**Review by gosonzhang — APPROVED** [review]

File: inlong-audit/audit-service/src/main/java/org/apache/inlong/audit/selector/impl/DBDataSource.java (hunk 1 above)
Comment (doleyzi) [reply inlong-audit/audit-service/src/main/java/org/apache/inlong/audit/selector/impl/DBDataSource.java:45]: These member variables will be updated, so there are no direct references to constants.

File: inlong-audit/audit-service/src/main/java/org/apache/inlong/audit/source/JdbcSource.java [hunk 2]
```diff
@@ -0,0 +1,306 @@
...
+                    data.setAuditId(AuditId);
+                    data.setAuditTag(AuditTag);
+                    data.setCount(count);
+                    data.setSize(size);
+                    data.setDelay(delay);
+                    dataQueue.push(data);
+                }
+                resultSet.close();
+                pstat.close();
+                connection.close();
```
Comment (aloyszhang) [inline inlong-audit/audit-service/src/main/java/org/apache/inlong/audit/source/JdbcSource.java:300]: Risk for resource leak, if an exception occurs before close.
Better to close in finally or use the try-with-resource to close resources automaticlly.

File: inlong-audit/audit-service/src/main/java/org/apache/inlong/audit/selector/impl/DBDataSource.java (hunk 1 above)
Comment (aloyszhang) [reply inlong-audit/audit-service/src/main/java/org/apache/inlong/audit/selector/impl/DBDataSource.java:45]: Got it.

File: inlong-audit/audit-service/src/main/java/org/apache/inlong/audit/source/JdbcSource.java [hunk 3]
```diff
@@ -0,0 +1,306 @@
...
+         */
+        public void statByStep() {
+            List<CompletableFuture<String>> futures = new ArrayList<>();
+            for (String auditId : auditIds) {
+                CompletableFuture<String> future = CompletableFuture.supplyAsync(() -> {
+                    aggregate(auditId);
+                    return "Task " + auditId + " completed";
+                });
+                futures.add(future);
+            }
```
Comment (aloyszhang) [inline inlong-audit/audit-service/src/main/java/org/apache/inlong/audit/source/JdbcSource.java:238]: 1. please  do the empty check for `auditIds` first
2. The future for each auditId does not need return a string, CompletableFuture<Void> is enough

File: inlong-audit/audit-service/src/main/java/org/apache/inlong/audit/source/JdbcSource.java [hunk 4]
```diff
@@ -0,0 +1,306 @@
...
+            CompletableFuture[] futureArray = futures.toArray(new CompletableFuture[futures.size()]);
+            CompletableFuture<Void> allFutures = CompletableFuture.allOf(futureArray);
+            allFutures.join();
```
Comment (aloyszhang) [inline inlong-audit/audit-service/src/main/java/org/apache/inlong/audit/source/JdbcSource.java:241]:
```suggestion
           CompletableFuture.allOf(futures.toArray(new CompletableFuture[futures.size()])).join();
```

**Review by justinwwhuang — DISMISSED** [review]

**Review by justinwwhuang — APPROVED** [review]

**Review by gosonzhang — APPROVED** [review]

**Pull request merged**